serving/src/main/java/ai/djl/serving/http/AdapterManagementRequestHandler.java

```diff
@@ -228,9 +233,11 @@ private void handleUpdateAdapter(
         }
 
         Adapter<Input, Output> adapter = modelInfo.getAdapter(adapterName);
+        String alias = req.headers().get("X-Amzn-SageMaker-Adapter-Alias");
 
         if (adapter == null) {
-            throw new BadRequestException(404, "The adapter " + adapterName + " was not found");
+            throw new BadRequestException(
+                    404, "The adapter " + (alias == null ? adapterName : alias) + " was not found");
```

Comment:
alias should always be provided, right? If so, we should validate. I know that behavior may change in the future, but for now I believe it is required. Correct me if i'm wrong